presto-main/src/main/java/com/facebook/presto/metadata/BuiltInFunctionNamespaceManager.java

```diff
@@ -697,6 +698,12 @@ public void createFunction(SqlInvokedFunction function, boolean replace)
         throw new PrestoException(GENERIC_USER_ERROR, format("Cannot create function in built-in function namespace: %s", function.getSignature().getName()));
     }
 
+    @Override
+    public void alterFunction(QualifiedFunctionName functionName, Optional<List<TypeSignature>> parameterTypes, AlterRoutineCharacteristics alterRoutineCharacteristics)
+    {
+        throw new PrestoException(GENERIC_USER_ERROR, format("Cannot alter function in built-in function namespace: %s", functionName));
```

Comment:
Hmm, just a thought, would people understand what is "built-in function namespace" intuitively? Or it's better to just say something like "alter function is not allowed for function xxx".

Reply:
When we have access control, `PERMISSION_DENIED` will be throw before reaching this line and this line should potentially be changed to an UnsupportedOperationException. I'll keep as is for now for consistency with createFunction and dropFunction.